niap-cc/KeyTestingPatches/Android16/system/vold/KeyStorage.cpp.patch

```diff
@@ -0,0 +1,28 @@
+--- KeyStorage.cpp.orig	2025-11-17 02:27:40.246730161 +0000
++++ KeyStorage.cpp	2025-11-19 09:50:15.685241084 +0000
+@@ -620,6 +620,25 @@
+             return false;
+         }
+     }
++
++    KeyBuffer hexKey;
++    StrToHex(*key, hexKey);
++    hexKey.push_back('\0');
++
++    std::string secDis;
++    StrToHex(secdiscardable_hash,secDis);
++    secDis.push_back('\0');
++
++    std::string appId2;
++    StrToHex(appId,appId2);
++    appId.push_back('\0');
```

Comment:
Should this have been `appId2` since `appId` was provided as input, and `appId2` is the one logged below?